qpx/cli/info.py

```
@@ -287,16 +281,14 @@ def _show_canonical_schema(structure: str):
     try:
         schema = cls.schema()
     except (AttributeError, TypeError) as e:
-        raise click.ClickException(
-            f"Cannot retrieve canonical schema for '{structure}': {e}"
-        )
+        raise click.ClickException(f"Cannot retrieve canonical schema for '{structure}': {e}")
```

Ruff (0.15.5)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)